install_config/topics/configuring_gce_objects.adoc

```diff
@@ -0,0 +1,176 @@
+documentation on external IP address] for more information.
+
+Cloud DNS::
+
+GCP cloud DNS is a DNS service used to publish domain names to the global
+DNS using GCP DNS servers. 
+
+The Public cloud DNS zone requires a domain name either purchased through
+Google’s "Domains" service or through a third-party provider. When you create the zone, you must add the name servers provided by Google to the registrar.
+
+See link:https://cloud.google.com/dns/overview[the GCP documentation on Cloud DNS] for more information.
+
+[NOTE]
+====
+GCP `VPC` networks have an internal DNS service that automatically
+resolves internal host names.
+
+The internal fully qualified domain name (FQDN) for an instance follows the `[HOST_NAME].c.[PROJECT_ID].internal` format.
+
+See link:https://cloud.google.com/compute/docs/internal-dns[the GCP documentation on Internal DNS] for more information.
+====
+
+Load balancing::
+
+The GCP load balancing service enables the distribution of traffic across multiple instances in the GCP cloud.
+
+There are five types of Load Balancing:
+
+* link:https://cloud.google.com/compute/docs/load-balancing/internal/[Internal]
+* link:https://cloud.google.com/compute/docs/load-balancing/network[Network load balancing]
+* link:https://cloud.google.com/compute/docs/load-balancing/http[HTTP(S) load balancing]
+* link:https://cloud.google.com/compute/docs/load-balancing/tcp-ssl/[SSL Proxy load balancing]
+* link:https://cloud.google.com/load-balancing/docs/tcp/[TCP Proxy load balancing]
+
+[NOTE]
+====
+HTTPS and TCP proxy load balancing are the only options for using
+HTTPS health checks for master nodes, which is checking the status of
+*_/healthz_*.
+
+Because HTTPS load balancing requires a custom certificate, this implementation
+uses TCP Proxy load balancing to simplify the process.
+====
+
+See link:https://cloud.google.com/compute/docs/load-balancing-and-autoscaling[the GCP documentation on Load balancing] for more information.
+
+Instances sizes::
+
+A successful {product-title} environment requires some minimum hardware
+requirements:
+
+.Instances sizes
+|===
+|Role |Size
+
+|Master |`n1-standard-8`
+|Node |`n1-standard-4`
+|===
+
+GCP allows you to create custom instance sizes to fit different requirements.
+See
+https://cloud.google.com/compute/docs/instances/creating-instance-with-custom-machine-type[Creating
+an Instance with a Custom Machine Type] for more information, or see
+link:https://cloud.google.com/compute/docs/machine-types[Machine types] and
+xref:../install/prerequisites.adoc#hardware[{product-title} Minimum Hardware
+Requirements] for more information about instance sizes.
+
+Storage Options::
+
+By default, each GCP instance has a small root persistent disk that contains the
+operating system. When applications running on the instance require more
+storage space, you can add additional storage options to the instance:
+
+* Standard Persistent Disks
```

Comment:
Do these terms use specific capitalization to match GCP usage? If not, I'd change it back to sentence case.